src/components/pages/dashboard/dashboard.js

```diff
@@ -144,6 +144,7 @@ class DashboardPage extends Component {
             <div className="last-refreshed-text"> {`${lang.LAST_REFRESHED} | `} </div>
             <div className="last-refreshed-time">{this.state.lastRefreshed.toLocaleString()}</div>
             <div onClick={this.refreshData} className="refresh-icon icon-sm" />
+            <div className="time-icon icon-sm" />
             <Select {...selectProps} />
```

Comment:
`select` should be indented

Reply:
Its indented.
![image](https://user-images.githubusercontent.com/16091688/33463024-eb55aede-d5ef-11e7-84aa-1309f1266ef5.png)